### ISSUE TITLE
Starting with v0.6.0, image tags now start with `v`

### ISSUE THREAD

**Issue by maelvls** [cert-manager/openshift-routes#75]
**TL;DR:** Starting with v0.6.0, all image tags will now contain a `v`.

As discussed in https://github.com/cert-manager/openshift-routes/pull/74#issuecomment-2238859842, the changes to the release process that occured in #60 changed the image tags. Before/after:

```diff
-ghcr.io/cert-manager/cert-manager-openshift-routes:0.5.0
+ghcr.io/cert-manager/cert-manager-openshift-routes:v0.6.0-alpha.0
```

All the other projects under the cert-manager org use a `v` prefix for image tags, so we are thinking of keeping this change.

This issue aims at letting everyone know. Feel free to comment on this issue if you feel this change will be breaking too much, or if you feel that a non-breaking way is possible.

FYI @vinny-sabatini @jacksgt @mbu147 @gaohoward @ctrought @hawksight

**Comment by hawksight** [comment]
As i am not personally using the project this seems fine to me. The consistency with other projects is a good thing.

Having this thread in the open is very helpful so I can share easily with consumers who do use this project thought 👍 .

Perhaps I'll make a community shout out too for increased awareness.

**Comment by inteon** [comment]
See https://github.com/cert-manager/openshift-routes/releases/tag/v0.6.0 for the release with the new image tag format.
